src/fsharp/vs/service.fs

```diff
@@ -760,6 +775,147 @@ type TypeCheckInfo
 
         let FindRecordFieldsInEnv(plid, residue) = GetClassOrRecordFieldsEnvironmentLookupResolutions(line, loc, plid, residue)
 
+        let getDeclaredItems isInRangeOperator =
```

Comment:
I'm wondering if you could submit the introduction of this function as a separate PR. Or else somehow avoid the more+indentation change.  Right now it's hard to spot what's changed in this (very subtle and fragile) code in this PR.